components/tabstrip/overview.md

```diff
@@ -10,35 +10,21 @@ position: 0
 
 # Blazor TabStrip Overview
 
-This article provides information about the <a href = "https://www.telerik.com/blazor-ui/tabstrip" target = "_blank">Blazor TabStrip component</a> and its core features.
-
->caption In this article:
-<!-- Start Document Outline -->
-* [Creating Blazor TabStrip](#creating-blazor-tabstrip)
-	* [Position of the Tabs](#position-of-the-tabs)
-	* [Persist Content](#persist-content)
-	* [Scrollable Tabs](#scrollable-tabs)
-	* [Styling and Appearance](#styling-and-appearance)
-* [Example](#example)
-* [See Also](#see-also)
-
-<!-- End Document Outline -->---
+The <a href = "https://www.telerik.com/blazor-ui/tabstrip" target = "_blank">Blazor TabStrip component</a> displays a collection of tabs, containing associated content, which enable the user to switch between different views inside a single component.
 
 ## Creating Blazor TabStrip
 
-To use a Telerik TabStrip for Blazor:
-
 1. Use the `<TelerikTabStrip>` tag.
 1. Add a nested `<TabStripTab>` tag for each tab you want to include in the component.
 1. Set tab titles through the `Title` parameter of the `<TabStripTab>` tags. You can additionally configure the tabs as desired - read more in the [Tabs Configuration]({%slug tabstrip-tabs-configuration%}) article.
 1. Place the desired content in the tabs - it can be any content, including other components.
 
->caption A Telerik TabStrip with example reference, tab position and disabled tab
+>caption TabStrip with specified tab position and a disabled tab
 
 ````CSHTML
 @using Telerik.Blazor.Components
 
-<TelerikTabStrip TabPosition="Telerik.Blazor.TabPosition.Left" @ref="myTabStrip">
+<TelerikTabStrip TabPosition="Telerik.Blazor.TabPosition.Left">
```

Comment:
Do we need non-default tab position at this stage of the getting started experience? If yes, we should mention it in the instructions.